src/Controls/StreetControls.js

```diff
@@ -102,19 +239,123 @@ class StreetControls extends FirstPersonControls {
     }
 
     /**
-     * Move the camera
+     * Method called when user click on the ground.</br>
+     * Note that this funtion contains default values that can be overrided, by overriding this function.
+     *
+     * @param {THREE.Vector3} position - The position
+     */
+    onClickOnGround(position) {
+        // compute a new position to go : 4 meter up from the picked position on the ground
+        const up = position.clone().normalize();
```

Comment:
does it work with the planar?

Reply:
Not yet, but I plan to add another example in planar. At this moment I will make all this work in planar view.

Reply:
Could you create a issue and add comment in the documentation?

Reply:
yes #1036